docs/integrations/app-integrations/slack.mdx

```diff
@@ -67,6 +67,7 @@ Here is how to set up a Slack app and generate both a Slack bot token and a Slac
         - im:write
         - mpim:read
         - users.profile:read
+        - users:read
```

Comment:
I've checked my permissions and I don't have 'users:read' but can read list of users

Reply:
I got an error that indicated that this permission is required. Let me mark it as optional then.